backend/api/constants.py

```diff
@@ -1,3 +1,6 @@
 class ApiExecution:
     PATH: str = "deployment/api"
     MAXIMUM_TIMEOUT_IN_SEC: int = 300  # 5 minutes
+    DEFAULT_TIMEOUT_IN_SEC: int = 80
+    FILES_FORM_DATA: str = "files"
+    TIMEOUT_FORM_DATA: str = "timeout"
```

Comment:
@muhammad-ali-e @chandrasekharan-zipstack should we use this constant here https://github.com/Zipstack/unstract/blob/779b944aab398b707cf71dd12d37058a2f5d1bdc/backend/api/serializers.py#L71